tests/about/about.spec.ts

```diff
@@ -1,7 +1,20 @@
 import { test, expect } from '@playwright/test';
 
+import { identifier } from '../../config';
+
+test.beforeEach(async ({ request, context }) => {
+  if(process.env.IS_REVIEW_APP === 'true') {
+    await context.addCookies([{
+      name: 'beta-access',
+      value: process.env.BETA_ACCESS_TOKEN || '',
+      path: '/',
+      domain: '.archive.org'
+    }]);
+  }
```

Comment:
QUESTION: This clause is repeated in every test file, which would make it hard to change if, for example, we needed to adjust it; we'd need to change it in many places. How can we extract it to some kind of mix-in or import or DRY it up in some other way?

Reply:
this is now refactored